rosbridge_library/src/rosbridge_library/capabilities/defragmentation.py

```diff
@@ -106,9 +106,7 @@ def defragment(self, message):
         msg_data = message.get("data")
 
         # Abort if any message field is missing
-        if ((msg_opcode == None) or (msg_id == None) or
-            (msg_num == None) or (msg_total == None) or
-            (msg_data == None)):
+        if None in (msg_opcode, msg_id, msg_num, msg_total, msg_data):
```

Comment:
Is this a common Python idiom? I haven't seen it before, but it does seem more concise than several `is None`.

Reply:
It is a pragmatic solution rather than a common solution.